test/e2e/multihoming_utils.go

```diff
@@ -358,7 +358,7 @@ func pingServer(clientPodConfig podConfiguration, serverIP string, args ...strin
 		clientPodConfig.name,
 		"--",
 		"ping",
-		"-c", "1", // send one ICMP echo request
+		"-c", "3", // send three ICMP echo requests
```

Comment:
are all these tests pings only? why do I feel like curl is a better test :) but I assume this existed even before you did tests right?

Reply:
I run curl first and then I send three pings :)